<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientForm</class>
 <widget class="QWidget" name="clientForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="send_label">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="rec_label">
     <property name="text">
      <string>接收</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTextEdit" name="sendEdit"/>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QTextEdit" name="recEdit_2"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="send_btn">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ip</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="ip_line"/>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLineEdit" name="port_line"/>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="client_btn">
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>port</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
